res/css/views/rooms/_EventTile.scss

```diff
@@ -737,11 +737,15 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
     align-items: center;
 
     &:hover,
-    // To cancel "&.mx_EventTile:hover .mx_EventTile_line"
+    // Override .mx_EventTile:not([data-layout=bubble]).mx_EventTile:hover .mx_EventTile_line
     &:not([data-layout=bubble]):hover .mx_EventTile_line {
         background-color: $system;
     }
 
+    &:not([data-layout=bubble]):hover .mx_EventTile_line {
```

Comment:
this selector looks identical to the one above?

Reply:
It might be hard to see, but the above has been combined with `&:hover` to ensure the same background-color.
This line is to avoid setting the box-shadow property to `&:hover`, in order to prevent a visual bug,

Reply:
It is really jarring to me how this selector is accomplishing this, given it doesn't mention threads at all, but it does empirically appear to do the right thing

Reply:
It is `.mx_EventTile[data-shape=ThreadsList]:not([data-layout=bubble]):hover .mx_EventTile_line`, so this line should be able to be `&:hover .mx_EventTile_line` if it helps as it does not matter whether it is the bubble layout or not.

If it would have not required the override, the selector above could be simpler.

Reply:
Thanks that cleared it up!